tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -101,15 +101,17 @@ def test_formdata_to_model_populates_collections_from_type_of_doc_thesis(u, ui):
 
 @mock.patch('inspirehep.modules.literaturesuggest.tasks.User')
 @mock.patch('inspirehep.modules.literaturesuggest.tasks.UserIdentity')
-def test_formdata_to_model_populates_collections_from_field_categories_if_arxiv(u, ui):
+def test_formdata_to_model_populates_collections_from_inspire_categories_if_arxiv(u, ui):
     data = {}
     extra_data = {}
     formdata = {
         'type_of_doc': 'foo',
         'title': [
             'bar',
         ],
-        'categories': 'baz',
+        'arxiv_eprints': {
+            'categories': ['baz']
+        },
```

Comment:
Here you are cheating: `formdata_to_model` can never have this argument, because such a structure is not part of the form data model. In fact, this is an instance of the problem I mentioned in https://github.com/inspirehep/inspire-next/pull/1885#discussion_r97901456: the DoJSON rules inside of `formdata_to_model` are conceptually wrong.